### DISCUSSION TITLE
"Сlawing back" status. How do I return the coins?

### DISCUSSION THREAD

**Discussion by SergeyLeee** [Chia-Network/chia-blockchain#18998] [Support]
Hello, I need help, I was making a transfer from chia's wallet to "Tangem" wallet. The translation was done with the claw back (revoke) the transaction function. The transfer has not arrived so far and I have made a refund of the transaction and now there is just a message hanging "!

**Comment by BrandtH22** [comment]
Hey @SergeyLeee , as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

Generally when clawbacks get stuck the best option is to force claim them and resend the transaction without a clawback. Also take note that tangem does not currently support clawbacks so you need to clawback the transaction and resend without a clawback. 

To do this you will need to use the CLI:

Get the clawback transaction id by running 
`chia wallet get_transactions --clawback`


Force the claim by running (replace TRANSACTION_ID with the tx id from above):
`chia rpc wallet spend_clawback_coins '{"coin_ids":["TRANSACTION_ID"], "force":true, "fee":0}'`

Let us know if you run into any issues or have questions running the commands

**Comment by BrandtH22** [comment]
No response from OP, please reopen the ticket if this issue persists
